### PR TITLE
Update new paths for package detection

### PR DESCRIPTION
**Pull request by mjcr99**
|Related issue|
|---|
|https://github.com/wazuh/wazuh/issues/25941|
|https://github.com/wazuh/wazuh/issues/25943|

## Description
Hi team, 

this PR updates the path used to look for packages as they have been updated in the wazuh/wazuh repository in this PR:
* https://github.com/wazuh/wazuh/pull/26236

### CODE REVIEW

**Review by MarcelKemp — APPROVED** [review]

File: src/vcpkg
Comment (jr0me) [inline src/vcpkg:1]: Was this change intentional?
